README.md

```diff
@@ -1 +1,49 @@
-# Weather_app
\ No newline at end of file
+# Weather app
+
+Weather app created using [openWeather](https://openweathermap.org/) API, Where users can access weather information about the location given in the top search bar.
+
+![screenshot](./src/weather_components/imgs/SCREENSHOT.png)
+
+
+## Built With
+
+- Javascript
+- Webpack
+- HTML
+- CSS 
+
+## Live Demo
+
+[Live Demo Link](https://shaqri.github.io/Weather_app/)
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+- Git clone https://github.com/Shaqri/Weather_app.git
+- Open the index.html inside the dist folder with your favorite browser
+- Explore the weather from all differents parts of the world you can imagine.
```

Comment:
## Required change
- Add additional instructions on how to;
- [ ] Install dependencies
- [ ] Setup users's  API keys in the environment 
- [ ] Run the application.